packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -543,7 +601,17 @@ class IOSDevice extends Device {
 
       _logger.printTrace('Application launched on the device. Waiting for Dart VM Service url.');
 
-      final int defaultTimeout = isWirelesslyConnected ? 45 : 30;
+      final int defaultTimeout;
+      if ((isCoreDevice || forceXcodeDebugWorkflow) && debuggingOptions.debuggingEnabled) {
+        // Core devices with debugging enabled takes longer because this
+        // includes time to install and launch the app on the device.
+        defaultTimeout = isWirelesslyConnected ? 75 : 60;
```

Comment:
curious what's typical xcode launch time? have you tried a few times to get some data on your laptop?

Reply:
It varies greatly, so it's hard to say what the "typical time" is. I have timed it, but it's a wide range. Sometimes it's like ~40 seconds, sometimes it's ~2 seconds. Can depend on if Xcode is loading device symbols, if the app is already installed already or not, etc.

Reply:
I think we can do longer time then. our machines are relatively new. they are just timeout, will stop early if found.

Reply:
I kept the time the same but I added a new user message below. More than a minute is a long time to go without any feedback so I think better to show message.